agent/check_handler.go

```diff
@@ -31,6 +31,11 @@ const (
 	measureNullStatus = "null-status"
 )
 
+var (
+	dupCheckRequestErr = errors.New("check request has already been received - agent and check may have multiple matching subscriptions")
```

Comment:
Nice 👌

Reply:
@asachs01 FYI ☝️ this is the new error message. 

Should we also change the log level? This is an agent log message, so not quite as problematic as backend logging. Let us know if we should also adjust the log level.

Reply:
The duplicate error is logged as a warning; old check request error is logged as an error. Let me know what y'all think.

Reply:
I think warn is appropriate @amdprophet. As for the error message, that looks great. 👍